packages/lab/src/list-next/ListNext.tsx

```diff
@@ -0,0 +1,170 @@
+   * Optional id prop
+   *
+   *The id for the list component
+   */
+  id?: string;
+  /**
+   * The number of items displayed in the visible area.
+   * Note that this determines the max height of the list.
+   */
+  displayedItemCount?: number;
+  // ListNextControlProps
+  onBlur?: FocusEventHandler;
+  onFocus?: FocusEventHandler;
+  onKeyDown?: KeyboardEventHandler;
+  onMouseDown?: MouseEventHandler;
+}
+
+export const ListNext = forwardRef<HTMLUListElement, ListNextProps>(
+  function ListNext(
+    {
+      children,
+      className,
+      disabled: listDisabled,
+      displayedItemCount: displayedItemCountProp = 4,
+      ListItem = DefaultListItem,
+      id: idProp,
+      onSelect,
+      onFocus,
+      onBlur,
+      onKeyDown,
+      onMouseDown,
+      style,
+      ...rest
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-next",
+      css: listNextCss,
+      window: targetWindow,
+    });
+
+    const childrenCount = Children.count(children);
+    const emptyList = childrenCount === 0;
+
+    const listId = useId(idProp) || "listNext"; // TODO: check why useId needs to return undefined
+
+    const displayedItemCount = Math.min(displayedItemCountProp, childrenCount);
+
+    const {
+      listRef,
+      focusedIndex,
+      selectedIndexes,
+      activeDescendant,
+      handleClick,
+      handleFocus,
+      handleKeyDown,
+      handleBlur,
+      handleMouseDown,
+    } = useList({
+      children,
+      displayedItemCount,
+    });
+
+    const forkedRef = useForkRef(ref, listRef);
+
+    const renderContent = () => {
+      return Children.map(children, (listItem, index) => {
+        if (!isValidElement(listItem)) return;
+        const { disabled: listItemDisabled, ...restListItemProps } =
+          listItem.props as ListNextProps;
+        const listItemProps = {
+          ...restListItemProps,
+          disabled: listItemDisabled || listDisabled,
+          onClick: (e: MouseEvent<HTMLUListElement>) => handleClick(e),
+          // focused is applicable for list items on focus using keyboard navigation only
+          focused: focusedIndex === index,
+          selected: selectedIndexes.includes(index),
+          id: `list-${listId}--list-item--${index}`,
+        };
+
+        return cloneElement(listItem, { ...listItemProps });
+      });
+    };
+    const listStyles = {
+      ...style,
+      "--listNext-displayedItemCount": displayedItemCount,
+    };
+
+    const activeElement = document.getElementById(activeDescendant);
+
+    const focusHandler = () => {
+      handleFocus();
+      onFocus && onFocus(activeElement);
+    };
+
+    const keydownHandler = (evt: KeyboardEvent) => {
+      handleKeyDown(evt);
+      onKeyDown && onKeyDown(activeElement);
+    };
+
+    const blurHandler = (evt: FocusEvent) => {
+      handleBlur();
+      onBlur && onBlur(evt);
+    };
+
+    const mousedownHandler = (evt: KeyboardEvent) => {
```

Comment:
```suggestion
    const mouseDownHandler = (evt: KeyboardEvent) => {
```
not major, but think it's good to align the casing, and same for keyDownHandler